NMS-14256: Additional XML Collector docs (#4652)
OpenNMS/opennms: docs/modules/reference/pages/configuration/performance-data-collection/collectors/xml.adoc

| How often to repeat the request in case of an error.
| 0

| disable-ssl-verification

> | Do not attempt to verify the name of on the SSL certificate against the `xml-source` URL attribute.

Reviewer: ```suggestion
| Do not attempt to verify the name on the SSL certificate against the `xml-source` URL attribute.
```
not sure if you prefer "on" or "of". 50/50 chance.

</xml-groups>
----

== HTTP headers

> If the endpoint being collected requires additional headers, they can be added as part of the `<request>` object.

Reviewer: ```suggestion
If the endpoint being collected requires additional headers, you can add them as part of the `<request>` object.
```
